azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/templates.json

```diff
@@ -451,6 +451,113 @@
+        "language": "Java"
+      },
+      "function": {
+        "disabled": false,
+        "bindings": [
+          {
+            "type": "daprBindingTrigger",
+            "bindingName": "scheduled",
+            "name": "triggerData",
+            "direction": "in"
+          },
+          {
+            "type": "daprPublish",
+            "direction": "out",
+            "name": "pubEvent",
+            "pubsubname": "messagebus",
+            "topic": "B"
+          },
+          {
+            "type": "daprTopicTrigger",
+            "pubsubname": "messagebus",
+            "topic": "A",
+            "name": "subEvent",
+            "direction": "in",
+            "dataType": "string"
+          }
+        ]
+      },
+      "files": {
+        "function.java": "package com.function;\r\n\r\nimport com.fasterxml.jackson.core.JsonProcessingException;\r\nimport com.fasterxml.jackson.databind.JsonNode;\r\nimport com.fasterxml.jackson.databind.ObjectMapper;\r\nimport com.microsoft.azure.functions.ExecutionContext;\r\nimport com.microsoft.azure.functions.OutputBinding;\r\nimport com.microsoft.azure.functions.annotation.FunctionName;\r\nimport com.microsoft.azure.functions.dapr.annotation.DaprStateOutput;\r\nimport com.microsoft.azure.functions.dapr.annotation.DaprTopicTrigger;\r\n\r\n\r\npublic class DaprTopicTriggerFunc {\r\n\r\n    \/*\r\n    Visit https:\/\/aka.ms\/azure-functions-dapr to learn how to use the Dapr extension.\r\n    These tasks should be completed prior to running :\r\n      1. Install Dapr\r\n      2. Change the bundle name in host.json to \"Microsoft.Azure.Functions.ExtensionBundle.Preview\" and the version to \"[4.*, 5.0.0)\"\r\n    Start function app with Dapr: dapr run --app-id functionapp --app-port 3001 --dapr-http-port 3501 --components-path .\/components\/ -- mvn azure-functions:run\r\n    Invoke function app: dapr publish --pubsub messagebus --publish-app-id functionapp --topic A --data '{\\\"value\\\": { \\\"orderId\\\": \\\"42\\\" } }'\r\n    *\/\r\n    @FunctionName(\"DaprTopicTriggerFunc\")\r\n    public String run(\r\n        @DaprTopicTrigger(\r\n            pubSubName = \"messagebus\",\r\n            topic = \"X\")\r\n            String request,\r\n        @DaprStateOutput(\r\n            stateStore = \"statestore\",\r\n            key = \"product\")\r\n        OutputBinding<String> product,\r\n        final ExecutionContext context) throws JsonProcessingException {\r\n        context.getLogger().info(\"Java DaprTopic trigger with DaprState output binding function processed a request from the Dapr Runtime.\");\r\n\r\n        \/\/ Get the CloudEvent data from the request body as a JSON string\r\n        ObjectMapper objectMapper = new ObjectMapper();\r\n        JsonNode jsonNode = objectMapper.readTree(request).get(\"data\");\r\n        String serializedJson = objectMapper.writeValueAsString(jsonNode);\r\n\r\n        product.setValue(serializedJson);\r\n\r\n        return serializedJson;\r\n    }\r\n}",
```

Comment:
please use `$packageName$` and `$className$` to replace the package name and class name

Reply:
It seems that we use `com.microsoft.azure.functions.dapr.*` in tempalte code, which should belong to `azure-functions-java-library-dapr`, we may also need to tell user introduce this dependency in pom.xml/build.gradle

Reply:
Here is the PR: https://github.com/microsoft/azure-maven-archetypes/pull/217/files

Reply:
@ASHIQUEMD Thanks for your efforts, I've added some comments to PR https://github.com/microsoft/azure-maven-archetypes/pull/217/files